en/docs/develop/customizations/customizing-the-developer-portal/customize-developer-portal-credentials.md

```diff
@@ -0,0 +1,103 @@
+# Customize Developer Portal Credential Preferences
+  
+You can change your WSO2 API Manager Developer Portal password, customize the default Developer Portal password policy, and enable or disable the Developer Portal password policy guidelines based on your requirement.
+
+## Change the Developer Portal password
+
+Follow the instructions below to change your password:
+
+1. Sign in to the Developer Portal.
+
+     `https://<hostname>:9443/devportal`
+
+     Example: `https://localhost:9443/devportal`
+
+     Use your username and password to sign in
+  
+2. Click on your username that appears at the top right corner and select **Change Password**.
+  
+     <img src="{{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png" alt="Change Developer Portal password User Menu" width="300px"/>
+  
+3. Enter your old password, a new password, re-enter the new password, and thereafter click **SAVE** to submit the changes. 
+
+     The new password should adhere to the custom password policies as described below.
+
+     <img src="{{base_path}}/assets/img/learn/change-devportal-password-submiting.png" alt="Developer portal password change submit" width="700"/>
+  
+## Customize Developer Portal password policy
+
+You can define your custom password policy by defining one or both of the followings
+    
+- [User store password RegEx](https://apim.docs.wso2.com/en/latest/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-jdbc-user-store/#configuring-a-jdbc-user-store)  
+
+     Example:
+
+     ```toml
+     [user_store]
+     type = "database_unique_id"
+
+     [user_store.properties]
+     PasswordJavaRegEx = "^[\\S]{6,30}$"
+     ```
+
+- [Identity management password policies](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/identity-management-for-the-api-dev-portal/#password-policies)
+     <html>    
+     <div class="admonition note">    
+     <p class="admonition-title">Note</p>    
+     <p>
+     <ul><li><p>
+     The password policy set by the Identity Management password policies can be overridden by the Management console for each tenant.
```

Comment:
Shall we change the above line as this. "The password policy set by identity management password policies through the configuration file can be overridden by Management console for each tenant."

Reply:
Fixed